handler/types.go

```diff
@@ -13,10 +13,10 @@ const (
 // BranchName assignment from gitlab repository
 const (
 	GROUPNAME   = "openebs"
-	MAYABRANCH  = "v0.9.x"
-	JIVABRANCH  = "v0.9.x"
-	ISTGTBRANCH = "v0.9.x"
-	ZFSBRANCH   = "v0.9.x"
+	MAYABRANCH  = "v1.0.x"
+	JIVABRANCH  = "v1.0.x"
+	ISTGTBRANCH = "v1.0.x"
+	ZFSBRANCH   = "v1.0.x"
```

Comment:
These branch are available in repository?

Reply:
yes ! Avaliable ...
![Screenshot from 2019-06-14 17-29-50](https://user-images.githubusercontent.com/40464957/59507943-0af6df80-8eca-11e9-94e4-8428c23c9f81.png)